webui/tests/cypress/integration/01-setup.spec.ts

```diff
@@ -1,4 +1,8 @@
 describe('setup', () => {
+  before(() => {
+    cy.login({ username: 'determined' });
```

Comment:
question: is `cy.login()` the same as `cy.login({ username: 'determined' })`?

Reply:
yeah ended up making the argument optional to avoid having to reference the default user everywhere. will update this